versions.tf

```diff
@@ -4,11 +4,11 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "3.94.0"
+      version = ">= 3.107.0"
     }
     azuread = {
       source  = "hashicorp/azuread"
-      version = "2.47.0"
+      version = ">= 2.47.0"
```

Comment:
We should move to v3 now

Reply:
azure erm updated one is at `4.8.0` shall we make a task for this or Should I do it together with my tf changes?